### PR TITLE
chore(react-scheduler): add the WeekView plugin API reference

### PR DESCRIPTION
**Pull request by MaximKudriavtsev**

### CODE REVIEW

File: packages/dx-react-scheduler/docs/reference/week-view.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+### Dependencies
+
+none
+
+### Properties
+
+Name | Type | Default | Description
+-----|------|---------|------------
+name? | string | `Week` | Specifies the view name. This is required property if you want to add many same plugins.
+excludedDays? | Array&lt;number&gt; | [] | Specifies days that is not displayed on the view.
```
Comment (SergeyAlexeev) [inline packages/dx-react-scheduler/docs/reference/week-view.md]: I think we can add the following note:
Sunday is 0, Monday is 1, and so on.

Comment (viterobk) [reply packages/dx-react-scheduler/docs/reference/week-view.md]: Agree. Done.

File: packages/dx-react-scheduler/docs/reference/week-view.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
+# WeekView Plugin Reference
```
Comment (SergeyAlexeev) [inline packages/dx-react-scheduler/docs/reference/week-view.md]: Too many empty rows.

File: packages/dx-react-scheduler/docs/reference/week-view.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+Field | Type | Description
+------|------|------------
+cellsData | Array&lt;Array&lt;[WeekView.CellData](#weekviewcelldata)&gt;&gt; | Specifies the cells meta data.
+tableRef | (ref: ReactInstance) => void | A function that accepts the table root React element.
+cellComponent | ComponentType&lt;[WeekView.TimeTableCellProps](#weekviewtimetablecellprops)&gt; | A component that renders a time table cell.
+rowComponent | ComponentType&lt;[WeekView.RowProps](#weekviewrowprops)&gt; | A component that renders a time table row.
+
+### WeekView.TimeTableCellProps
+
+Describes properties passed to a component that renders a time scale cell.
```
Comment (SergeyAlexeev) [inline packages/dx-react-scheduler/docs/reference/week-view.md]: 'time scale cell' => 'time **table** cell'

File: packages/dx-react-scheduler/docs/reference/week-view.md [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+
+Field | Type | Description
+------|------|------------
+startDate | Date | Specifies the cell a start time.
+endDate | Date | Specifies the cell end time.
```
Comment (SergeyAlexeev) [inline packages/dx-react-scheduler/docs/reference/week-view.md]: These props aren't required. What about the `children` prop?

File: packages/dx-react-scheduler/docs/reference/week-view.md [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+
+## Import
+
+Use the following statement to import a plugin with embedded theme components:
+
+```js
+import { WeekView } from '@devexpress/dx-react-scheduler-material-ui';
+```
+
+If you want to use custom components, you can import the themeless plugin:
```
Comment (dirk-pieterse) [inline packages/dx-react-scheduler/docs/reference/week-view.md]: Same change here

**Pull request merged**